### DISCUSSION TITLE
Frameless window in native mode can not be minimized by clicking its taskbar button

### DISCUSSION THREAD

**Discussion by Aues6uen11Z** [zauberzeug/nicegui#2838] [Q&A] [answered]
### Question

Hi, I found that in native mode, a borderless window cannot be minimized by clicking its Windows taskbar icon, and this issue can be easily replicated as below. 

```python
from nicegui import ui

ui.label('Frameless app running in native mode')
ui.run(native=True, frameless=True)
```

I also noticed that both Qt and pywebview exhibit the same problem, with Qt already having a solution in place. Thus, I wonder if NiceGUI and pywebview offer similar workarounds. After searching, I haven't come across anyone reporting this issue, so I'd appreciate any assistance on this matter. Many thanks in advance!

**Comment by falkoschindler** [comment, marked as answer]
Hi @Aues6uen11Z,

Do I understand correctly that a simple pywebview app has the same problem? (I can't try it myself, since I don't have access to a Windows machine.)

```py
import webview

webview.create_window('Hello world', 'https://pywebview.flowrl.com/', frameless=True)
webview.start()
```

If that's the case, I recommend filing an issue at https://github.com/r0x0r/pywebview.

**Reply by Aues6uen11Z** [reply]
Yes, a simple pywebview app has the same problem on Windows machine. What I originally intended to ask was whether NiceGUI has its own solution, given that it doesn't directly use pywebview, and there might be differences between them. But you're right, I should start by filing an issue to pywebview first. Thank you for your insight.😃

**Reply by rodja** [reply]
We are directly using pywebview. So this should be handled upstream.
